<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WalletdParamsDialog</class>
 <widget class="QDialog" name="WalletdParamsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wallet daemon parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Bytecoin node</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="radioBuiltinBytecoind">
            <property name="text">
             <string>bui&amp;lt-in bytecoin node</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioExternalBytecoind">
            <property name="text">
             <string>e&amp;xternal bytecoin node</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="1" column="0">
             <widget class="QLineEdit" name="editHost">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>127.0.0.1:8081</string>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>127.0.0.1:8081</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Host:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Net</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioMainNet">
          <property name="text">
           <string>&amp;mainnet</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioStageNet">
          <property name="text">
           <string>&amp;stagenet</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioTestNet">
          <property name="text">
           <string>&amp;testnet</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Additional command line options:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="paramsEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>WalletdParamsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>851</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>WalletdParamsDialog</receiver>
   <slot>applyParams()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>WalletdParamsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioExternalBytecoind</sender>
   <signal>toggled(bool)</signal>
   <receiver>editHost</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editHost</sender>
   <signal>textChanged(QString)</signal>
   <receiver>WalletdParamsDialog</receiver>
   <slot>hostChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioExternalBytecoind</sender>
   <signal>toggled(bool)</signal>
   <receiver>WalletdParamsDialog</receiver>
   <slot>externalBytecoindToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveParams()</slot>
  <slot>applyParams()</slot>
  <slot>hostChanged()</slot>
  <slot>externalBytecoindToggled(bool)</slot>
 </slots>
</ui>
